app/javascript/src/NewApplication/components/BuyerSelect.jsx

```diff
@@ -2,45 +2,54 @@
 
 import * as React from 'react'
 
+import { sortable } from '@patternfly/react-table'
+import { fetchPaginatedBuyers } from 'NewApplication'
 import { SelectWithModal } from 'Common'
 
 import type { Buyer } from 'NewApplication/types'
 
 type Props = {
   buyer: Buyer | null,
-  buyers: Buyer[],
+  mostRecentlyCreatedBuyers: Array<Buyer>,
+  buyersCount: number,
   onSelectBuyer: (Buyer | null) => void,
+  buyersPath: string,
   isDisabled?: boolean
 }
 
-const BuyerSelect = ({ buyer, buyers, onSelectBuyer, isDisabled }: Props): React.Node => {
-  const cells = [
-    { title: 'Name', propName: 'name' },
-    { title: 'Admin', propName: 'admin' },
-    { title: 'Signup date', propName: 'createdAt' }
-  ]
+const cells = [
+  { title: 'Name', propName: 'name' },
+  { title: 'Admin', propName: 'admin' },
+  { title: 'Signup date', propName: 'createdAt', transforms: [sortable] }
+]
 
-  return (
-    // $FlowFixMe[prop-missing] Implement async pagination
-    <SelectWithModal
-      label="Account"
-      fieldId="account_id"
-      id="account_id"
-      name="account_id"
-      // $FlowIgnore[incompatible-type] Buyer implements Record
-      item={buyer}
-      items={buyers.map(b => ({ ...b, description: b.admin }))}
-      itemsCount={buyers.length}
-      cells={cells}
-      // $FlowIssue[incompatible-type] It should not complain since Record.id has union "number | string"
-      onSelect={onSelectBuyer}
-      header="Most recently created Accounts"
-      isDisabled={isDisabled}
-      title="Select an Account"
-      placeholder="Select an Account"
-      footerLabel="View all Accounts"
-    />
-  )
-}
+const BuyerSelect = ({
+  buyer,
+  mostRecentlyCreatedBuyers,
+  buyersCount,
+  onSelectBuyer,
+  buyersPath,
+  isDisabled
+}: Props): React.Node => (
+  // $FlowIssue[incompatible-type-arg] id can be string too
+  <SelectWithModal
+    label="Account"
+    fieldId="account_id"
+    id="account_id"
+    name="account_id"
+    item={buyer}
+    items={mostRecentlyCreatedBuyers}
+    itemsCount={buyersCount}
+    cells={cells}
+    onSelect={onSelectBuyer}
+    fetchItems={(params) => fetchPaginatedBuyers(buyersPath, params)}
+    onAbortFetch={() => console.log('abort')}
```

Comment:
There is no `onAbortFetch` prop in https://github.com/3scale/porta/blob/2.11/THREESCALE-7030_new_app_typeahead/app/javascript/src/Common/components/SelectWithModal.jsx#L32

And `console.log` will make lint fail